spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -86,33 +84,14 @@ def get_parser():
     mandatory.add_argument(
         '-l',
         metavar=Metavar.file,
-        required=True,
         help=textwrap.dedent("""
             NIfTI file that includes labels at the compression sites. Each compression site is denoted by a single voxel of value `1`. Example: `sub-001_T2w_compression_labels.nii.gz`
 
             Note: The input and the compression label file must be in the same voxel coordinate system and must match the dimensions between each other.
         """),  # noqa: E501 (line too long)
     )
-    mandatory.add_argument(
-        '-metric',
-        required=False,
-        help='Metric to normalize. ',
-        default='diameter_AP',
-        choices=['diameter_AP', 'area', 'diameter_RL', 'eccentricity', 'solidity'],
-    )
-    mandatory.add_argument(
-        '-normalize-hc',
-        metavar=Metavar.int,
-        type=int,
-        choices=[0, 1],
-        help=textwrap.dedent("""
-            Set to 1 to normalize the metrics using a database of healthy controls. Set to 0 to not normalize.
 
-            Note: This flag should not be set to 1 when computing the MCC (i.e. using spinal canal segmentation). It should only be used when computing the MSCC (i.e. using spinal cord segmentation).
-        """),  # noqa: E501 (line too long)
-    )
-
-    optional = parser.add_argument_group("OPTIONAL ARGUMENTS")
```

Comment:
Good catch, finding these non-mandatory arguments that were under `MANDATORY`!

I also completely agree with moving them into their own `NORMALIZATION` category, feels very clean. :)